controls/def.cf

```diff
@@ -349,7 +349,8 @@ bundle common inventory_control
 # are disabled.
 {
   vars:
-      "lldpctl_exec" string => "/usr/bin/lldpctl";
```

Comment:
@BasementTrix I added a case for `/usr/local/bin/lldpctl` as follows:

* if `/usr/sbin/lldpctl` exists, use it
* else if `/usr/local/bin/lldpctl` exists, use it
* else use `/usr/sbin/lldpctl` as a last resort

Try it out and see if it does the right thing for you. Thanks.